store/sqlstore/session_store.go

```diff
@@ -106,34 +106,28 @@ func (me SqlSessionStore) Get(sessionIdOrToken string) store.StoreChannel {
 	})
 }
 
-func (me SqlSessionStore) GetSessions(userId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var sessions []*model.Session
-
-		tcs := me.Team().GetTeamsForUser(userId)
```

Comment:
Here we were running the `GetTeamsForUser` in async mode, and during that returns, we were running the Select. With your change, you are doing that one after the another instead of parallel. I would expect to keep this call here as was before. and read the channel after the "Select".